Fst2List.cpp

```diff
@@ -270,8 +284,11 @@ class CFstApp {
   char ofExt[16];
   char ofnameOnly[512];        // output file name
   char defaultIgnoreName[512]; // input file name
-  bool inMorphoMode = false;  // true if the current state is in morphological mode
-  bool isKorean = false;
+  bool inMorphoMode;  // true if the current state is in morphological mode
+  bool isKorean;
+  struct locate_parameters* p;
+  int morphDicCnt;  // number of dic to explore when a lexical mask is encoutered
+  bool isMdg;
```

Comment:
Comment about this variable (for morphological dictionary-graph)

Reply:
done